### ISSUE TITLE
3.52 and 3.53 not uploaded to S3?

### ISSUE THREAD

**Issue by henderea** [rubinius/homebrew-apps#4]
The most recent 2 versions of the homebrew formula fail to install because the URL doesn't exist in S3.  I checked the history in this repo, and 3.51 downloads just fine in my browser, but 3.52 and 3.53 both give error pages.  The formula also seems to be missing the SHA for 3.52 and 3.53.

Currently, I'm unable to upgrade rubinius past 3.51 on homebrew.  Can you please try to fix the issue?

**Comment by olistik** [comment]
@henderea given a newer version (3.54) is out I've created #5

**Comment by colstrom** [comment]
Confirmed. Versions after `3.51` return `404` responses from S3, using the URLs provided.

Equally concerning, the checksums have been removed after that point. Can you shed some light on that, @brixen? I assume there's a good story behind removing verification from executable code fetched from the internet. ^_~

**Comment by colstrom** [comment]
Good catch on the URL changes, @olistik! I've applied the same process to `3.55`, yielding #6.
